--- maven-plugin-testing-harness/src/main/java/org/apache/maven/plugin/testing/SilentLog.java.orig	2010-09-17 17:21:00.628814598 +0800
+++ maven-plugin-testing-harness/src/main/java/org/apache/maven/plugin/testing/SilentLog.java	2010-09-17 17:22:11.113099358 +0800
@@ -325,4 +325,10 @@
     {
         return null;
     }
+
+    public void setThreshold( int currentThreshold )
+    {
+        //this.currentThreshold = currentThreshold;
+    }
+
 }
